lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -263,10 +271,12 @@ class ImageElements extends Gatherer {
       const matchedRules = await driver.sendCommand('CSS.getMatchedStylesForNode', {
         nodeId: nodeId,
       });
-      const sourceWidth = getEffectiveSizingRule(matchedRules, 'width');
-      const sourceHeight = getEffectiveSizingRule(matchedRules, 'height');
-      const sourceRules = {cssWidth: sourceWidth, cssHeight: sourceHeight};
-      Object.assign(element, sourceRules);
+      const width = getEffectiveSizingRule(matchedRules, 'width');
+      const height = getEffectiveSizingRule(matchedRules, 'height');
+      // COMPAT: Maintain backcompat for <= 7.0.1
+      element.cssWidth = width === null ? undefined : width;
+      element.cssHeight = height === null ? undefined : height;
+      Object.assign(element, {_privateCssSizing: {width, height}});
```

Comment:
```suggestion
      element._privateCssSizing = {width, height};
```

(not actually important, feel free to disregard :) just wondering why we used `Object.assign()` here. Maybe the impl changed over time and this kept tsc happy at some point?)

Reply:
yeah no idea, but i'll change this and also the equivalent one in fetchElementWithSizeInformation